lib/vrl/parser/src/arbitrary.rs

```diff
@@ -262,6 +262,7 @@ impl<'a> ArbitraryDepth<'a> for FunctionCall {
             ident,
             abort_on_error,
             arguments,
+            closure: None,
```

Comment:
This is fine for now, but we really need to make sure that we don't forget to fully implement this prior to release so that the VM and AST versions are fully fuzzed. Might be worth raising an issue to ensure it gets done.

Reply:
Filed #12366